src/Xamarin.Android.Build.Tasks/Tasks/ConvertDebuggingFiles.cs

```diff
@@ -28,7 +29,7 @@ public override bool RunTask ()
 					continue;
 
 				try {
-					MonoAndroidHelper.SetWriteable (pdb);
+					Microsoft.Android.Build.Tasks.Files.SetWriteable (pdb);
```

Comment:
does this need to be fully qualified?

Reply:
This property was interfering: https://github.com/xamarin/xamarin-android/blob/060be4b6fdc040ce7a9af238d5b28af624cea46e/src/Xamarin.Android.Build.Tasks/Tasks/ConvertDebuggingFiles.cs#L17